test/runtests.jl

```diff
@@ -47,3 +47,54 @@ end
   JtF = jtprod_residual(nls_model, x, F)
   @test all(g .≈ JtF)
 end
+
+@testset "SVM-Train" begin
+  nlp_train,nls_train,sol = svm_train_model()
+  @test typeof(nlp_train) <: ADNLPModel
+  @test typeof(nls_train) <: ADNLSModel
+  @test typeof(sol) == Vector{Float64}
+  ### below is TBD
+  # @test model.meta.nvar == 5
+  # @test model.nls_meta.nequ == 202
+  # @test all(model.meta.x0 .== 1)
+  # @test length(findall(x -> x .!= 0, sol)) == 2
+end
+
+@testset "SVM-Test" begin
+  nlp_test,nls_test,sol = svm_test_model()
+  @test typeof(nlp_test) <: ADNLPModel
+  @test typeof(nls_test) <: ADNLSModel
+  @test typeof(sol) == Vector{Float64}
+  ### Below is TBD
+  # @test model.meta.nvar == 5
+  # @test model.nls_meta.nequ == 202
+  # @test all(model.meta.x0 .== 1)
+  # @test length(findall(x -> x .!= 0, sol)) == 2
+end
+
+##### Obsolete since svm_train/test_model() does not yield resid() anymore
```

Comment:
Which `resid()` are you referring to here? There's an NLSModel with a `resid()` in there.

Reply:
Not sure what this is referring to - I think I was outputting the residual to compare with your AD derivative tests. We can omit this if you wish.

Reply:
I'm just not understanding which `resid!()` method the comment is referring to.

Reply:
It was an old function I was using to test against AD. It doesn't exist anymore.